chained_filter_controller/CMakeLists.txt

```diff
@@ -0,0 +1,107 @@
+# cmake_minimum_required and project setup
+cmake_minimum_required(VERSION 3.8)
+project(chained_filter_controller)
+
+# Enable warnings
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# Dependencies
+find_package(ament_cmake REQUIRED)
+find_package(controller_interface REQUIRED)
+find_package(filters REQUIRED)
+find_package(pluginlib REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(rclcpp_lifecycle REQUIRED)
+find_package(hardware_interface REQUIRED)
+find_package(generate_parameter_library REQUIRED)
```

Comment:
please have a look at the other packages to follow the same style (look for `THIS_PACKAGE_INCLUDE_DEPENDS`)